src/components/Card/Card.tsx

```diff
@@ -0,0 +1,15 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { FC, ReactChild } from "react";
+
+export type CardProps = {
+    children?: ReactChild;
+};
```

Comment:
```suggestion
// import { PropsWithChildren } from 'react';
export type CardProps = PropsWithChildren<{}>
```

Reply:
Thanks, @SamuelAlev! This could definitely be done when there are more props in the future. Currently, eslint doesn't like it empty.